armsrc/flashmem.c

```diff
@@ -104,24 +153,21 @@ void FlashStop(void) {
 
 //	send one byte over SPI
 uint16_t FlashSendByte(uint32_t data) {	
-	uint16_t incoming = 0;
-
-	WDT_HIT();
 
 	// wait until SPI is ready for transfer
-	while ((AT91C_BASE_SPI->SPI_SR & AT91C_SPI_TXEMPTY) == 0) {};
+	//if you are checking for incoming data returned then the TXEMPTY flag is redundant
+	//while ((AT91C_BASE_SPI->SPI_SR & AT91C_SPI_TXEMPTY) == 0) {};
```

Comment:
remove?

Reply:
Yes and no, there seems to be silicon bugs with TDRE and I tend to ask for more testing before removing this definitely.

Reply:
ok